### PR TITLE
made the api wrapper a thing now

### PR DESCRIPTION
**Pull request by ConnorDoesDev**

### CODE REVIEW

**Comment by ConnorDoesDev** [comment]
bonk

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "botapi",
+    "delbotapi"
+  ],
+  "author": "Connor!#0800",
+  "license": "OSL-3.0",
+  "bugs": {
+    "url": "https://github.com/discordextremelist/del.js/issues"
+  },
+  "homepage": "https://github.com/discordextremelist/del.js#readme",
+  "dependencies": {
```
Comment (Kyro3400) [inline package.json:36]: Shouldn't you also add `discord.js` and `del.js` as a package so people won't have to worry about it?

**Review by Kyro3400 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ConnorDoesDev) [reply package.json:36]: what? that would've make absolutely no sense

Comment (Kyro3400) [reply package.json:36]: ok

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
-# del-api-js
-The official JS library for the Discord Extreme List API.
+# DiscordExtremeList's API Wrapper
+
+## Getting Started
+
+#### Importing and "initializing" the package
+```js
+const DEL = require('del.js') // importing del
+const del = new DEL("auth token", "client id") // initializing it
```
Comment (J-Human) [inline README.md]:
```suggestion
const del = new DEL('auth token', 'client id'); // initializing it
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
...
+|--------------|----------|--------------|--------------|
+key | String | false | The API Token found on your bot's page.
+id | Snowflake | false | Your bot's client ID.
+
+--- 
+
+### example
+
+```js
+const Discord = require("discord.js")
```
Comment (J-Human) [inline README.md:22]:
```suggestion
const Discord = require('discord.js');
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,36 @@
...
+const client = new Discord.Client()
+const prefix = "del.";
```
Comment (J-Human) [inline README.md:24]:
```suggestion
const prefix = 'del.';
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,36 @@
...
+const DEL = require("del.js")
```
Comment (J-Human) [inline README.md:25]:
```suggestion
const DEL = require('del.js');
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,36 @@
...
+const del = new DEL("auth token", client.user.id,)
```
Comment (J-Human) [inline README.md:26]:
```suggestion
const del = new DEL('auth token', client.user.id);

File: README.md [hunk 7]
```diff
@@ -1,2 +1,36 @@
...
+
+client.on("ready", () => {
```
Comment (J-Human) [inline README.md:28]:
```suggestion
client.on('ready', () => {
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,36 @@
...
+console.log(`Logged in as ${client.user.tag}.`)
```
Comment (J-Human) [inline README.md:29]:
```suggestion
  console.log(`Logged in as ${client.user.tag}.`);
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,36 @@
...
+setInterval(() => {
```
Comment (J-Human) [inline README.md:30]:
```suggestion
  setInterval(() => {
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,36 @@
...
+   del.post(client.guilds.cache.size, 0) // You will probably need to change this.
+  }, 600000) // 10 minutes in Milliseconds
```
Comment (J-Human) [inline README.md:32]:
```suggestion
  }, 600000); // 10 minutes in milliseconds
```

File: README.md [hunk 11]
```diff
@@ -1,2 +1,36 @@
...
+});
+
+client.login("token")
```
Comment (J-Human) [inline README.md:35]:
```suggestion
client.login('token');
```

File: README.md [hunk 12]
```diff
@@ -1,2 +1,36 @@
...
+const Discord = require("discord.js")
+const client = new Discord.Client()
+const prefix = "del.";
+const DEL = require("del.js")
+const del = new DEL("auth token", client.user.id,)
+
+client.on("ready", () => {
+console.log(`Logged in as ${client.user.tag}.`)
+setInterval(() => {
+   del.post(client.guilds.cache.size, 0) // You will probably need to change this.
```
Comment (J-Human) [inline README.md:31]:
```suggestion
   del.post(client.guilds.cache.size, 0); // You will probably need to change this.
```

File: package.json [hunk 13]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "del.js",
+  "version": "1.0.0",
+  "description": "The API Wrapper for Discord's unbiased list, giving bots and servers of all sizes a big chance!",
```
Comment (J-Human) [inline package.json:4]:
```suggestion
  "description": "The API wrapper for Discord's unbiased list, giving bots and servers of all sizes a big chance!",
```

File: README.md [hunk 14]
```diff
@@ -1,2 +1,36 @@
-# del-api-js
-The official JS library for the Discord Extreme List API.
+# DiscordExtremeList's API Wrapper
+
+## Getting Started
+
+#### Importing and "initializing" the package
+```js
+const DEL = require('del.js') // importing del
```
Comment (J-Human) [inline README.md]:
```suggestion
const DEL = require('del.js'); // importing del
```

**Review by J-Human — CHANGES_REQUESTED** [review]
Various consistency fixes:
* Indentation (not quite sure so I assumed 2).
* Semicolons.
* Single quotes (as seen in the API wrapper file, so do the same with the sample).

Capitalization fixes, too.

P.S.: Maybe we should have an Eris sample, too?

**Pull request merged**